src/SDKs/Search/DataPlane/Microsoft.Azure.Search.Data/Customizations/Documents/DocumentsOperations.cs

```diff
@@ -85,23 +85,28 @@ internal DocumentsOperations(SearchIndexClient client)
                     search,
                     suggesterName,
                     searchRequestOptions,
-                    autocompleteParametersPayload,
+                    autocompleteParameters,
                     customHeaders,
                     cancellationToken);
             }
             else
             {
+                string searchFieldsStr = null;
+                if (autocompleteParameters != null && autocompleteParameters.SearchFields != null)
```

Comment:
nit: You can shorten this to `if (autocompleteParameters?.SearchFields != null)`

Reply:
Thanks!